.github/workflows/cicd.yml

```
@@ -0,0 +1,62 @@
+          echo "${{ secrets.APPLICATION_YML }}" > src/main/resources/application.yml
+          echo "${{ secrets.APPLICATION_DEV_YML }}" > src/main/resources/application-dev.yml
+
+      # Gradle 빌드
+      - name: Build with Gradle
+        run: |
+          chmod +x ./gradlew
+          ./gradlew clean build -x test
+
+      # Docker Hub 로그인
+      - name: Docker Login
+        uses: docker/login-action@v2
```

actionlint (1.7.8)

the runner of "docker/login-action@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)